### PR TITLE
Add Arbitrum Sepolia

### PR DESCRIPTION
**Pull request by a-thomas-22**
Safe deployment PR: https://github.com/safe-global/safe-deployments/pull/290

Also removed some deprecated arbitrum related blockscout block explorers

### CODE REVIEW

**Comment by a-thomas-22** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by a-thomas-22** [comment]
recheck

File: gnosis/safe/safe_deployments.py [hunk 1]
```diff
@@ -284,6 +284,7 @@
             "333999": "0xA238CBeb142c10Ef7Ad8442C6D1f9E89e07e7761",
             "421611": "0xA238CBeb142c10Ef7Ad8442C6D1f9E89e07e7761",
             "421613": "0xA238CBeb142c10Ef7Ad8442C6D1f9E89e07e7761",
+            "421614": "0xA238CBeb142c10Ef7Ad8442C6D1f9E89e07e7761",
```
Comment (moisses89) [inline gnosis/safe/safe_deployments.py:287]: safe_deployments.py was updated [here](https://github.com/safe-global/safe-eth-py/pull/674/files#diff-3cec09db1d13253c39201c07d5f0397355176f99279adc5155d5071982f9cf3dR324)  using the generator script. 
Could you remove your changes on this file?

File: gnosis/eth/clients/blockscout_client.py [hunk 2]
```diff
@@ -48,9 +48,6 @@ class BlockscoutClient:
         EthereumNetwork.EVMOS_TESTNET: "https://evm.evmos.dev/graphiql",
         EthereumNetwork.KCC_MAINNET: "https://scan.kcc.io/graphiql",
         EthereumNetwork.KCC_TESTNET: "https://scan-testnet.kcc.network/graphiql",
-        EthereumNetwork.ARBITRUM_ONE: "https://explorer.arbitrum.io/graphiql",
-        EthereumNetwork.ARBITRUM_NOVA: "https://nova-explorer.arbitrum.io/graphiql",
-        EthereumNetwork.ARBITRUM_GOERLI: "https://goerli-rollup-explorer.arbitrum.io/graphiql",
```
Comment (moisses89) [inline gnosis/eth/clients/blockscout_client.py:53]: Why are you removing other blockscout chains different to Arbitrum Sepolia?

File: gnosis/safe/addresses.py [hunk 3]
```diff
@@ -181,6 +181,10 @@
         ("0x3E5c63644E683549055b9Be8653de26E0B4CD36E", 11545, "1.3.0+L2"),
         ("0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552", 11546, "1.3.0"),
     ],
+    EthereumNetwork.ARBITRUM_SEPOLIA: [
+        ("0x3E5c63644E683549055b9Be8653de26E0B4CD36E", 154, "1.3.0+L2"),
+        ("0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552", 155, "1.3.0"),
+    ],
```
Comment (moisses89) [inline gnosis/safe/addresses.py:187]: Please add also ProxyFactory address and block on PROXY_FACTORIES section https://github.com/safe-global/safe-eth-py/pull/669/files#diff-62bfa04ec3dcf06fa5a67465980b383d49368b25afe14cc3abb576b003d25062R604

**Review by moisses89 — CHANGES_REQUESTED** [review]

File: gnosis/safe/safe_deployments.py (hunk 1 above)
Comment (a-thomas-22) [reply gnosis/safe/safe_deployments.py]: Removed

File: gnosis/eth/clients/blockscout_client.py (hunk 2 above)
Comment (a-thomas-22) [reply gnosis/eth/clients/blockscout_client.py:53]: These endpoints are not available and we (Offchain Labs) will not be bringing them back. The one blockscout instance still online(Nova) has the graphiql endpoint disabled. Do you want this in a different PR?

File: gnosis/safe/addresses.py (hunk 3 above)
Comment (a-thomas-22) [reply gnosis/safe/addresses.py:187]: Done

File: gnosis/eth/clients/blockscout_client.py (hunk 2 above)
Comment (moisses89) [reply gnosis/eth/clients/blockscout_client.py:53]: No, it is ok then.

**Review by moisses89 — APPROVED** [review]

**Comment by moisses89** [comment]
@a-thomas-22 I didn't see the missing chain_id :smiling_face_with_tear:  could you add it here? https://github.com/safe-global/safe-eth-py/blob/master/gnosis/eth/ethereum_network.py#L647

**Review by moisses89 — CHANGES_REQUESTED** [review]
@a-thomas-22 I didn't see the missing chain_id 🥲 could you add it here? https://github.com/safe-global/safe-eth-py/blob/master/gnosis/eth/ethereum_network.py#L647

**Comment by moisses89** [comment]
> @a-thomas-22 I didn't see the missing chain_id 🥲 could you add it here? https://github.com/safe-global/safe-eth-py/blob/master/gnosis/eth/ethereum_network.py#L647

@a-thomas-22 Have you any chance to review this?

**Review by moisses89 — APPROVED** [review]

**Pull request merged**